tests/spec/fs.readdir.spec.js

```diff
@@ -55,3 +55,25 @@ describe('fs.readdir', function() {
     });
   });
 });
+
+/**
+ * fsPromises tests
+ */
+
+describe('fsPromises.readdir', function() {
```

Comment:
Move the following test into the `describe('fs.readdir'...)` block.  This makes it easier to add more related tests if others are working on the same function. You may also need to change the description of each test to separate them from the original test. e.g. prefix them with `(promises)`

Reply:
I just think it would be better organized to separate the function callback and promises-based tests. I've looked over other pull requests and they seem to follow the same pattern. For example, `fs.promises.appendFile`: https://github.com/filerjs/filer/pull/401/commits/24b9dc7b457feb1dd4a87aa54ed51a47b143bbad and `fs.promises.stat`: https://github.com/filerjs/filer/pull/406/commits/d07080c75c1db06db20ba4263a1bc147ba685246